build.py

```diff
@@ -2639,9 +2661,12 @@ def enable_all():
                 components,
                 backends,
             )
-
         # Commands to build each backend...
         for be in backends:
+            # Define fucntion to create cmake_script with backend name as suffix
```

Comment:
Make sure to do `pre-commit install` locally in the repo and possibly `pre-commit run --all-files` so it can locally fix the issues being reported by the :x: